components/header/Header.js

```diff
@@ -17,12 +17,8 @@ const Header = ({ children, bgImg }) => {
   };
 
   const handleCloseNavMenu = (page) => {
-    if (page.text === 'Home') {
-      router.push('/')
-    } else if (page.text === 'Careers') {
-      router.push('/careers')
-    } else if (page.text === 'Contact Us') {
-      router.push('/contact-us')
+    if(page.path !== router.path) {
```

Comment:
@uzairmirza90 why have you changed this?

Reply:
Can you please do that in separate PR? We will have to verify the logic. It will delay the merge of this PR. I have created an issue for dynamic routing https://github.com/prixite/prixite.github.io/issues/57.